### DISCUSSION TITLE
09_csv_to_points

### DISCUSSION THREAD

**Discussion by coblinski** [opengeos/leafmap#74] [Q&A] [answered]
I am trying to use my files to test the codes, but I got this error on image :
It is weird, because the names are similar.
![123](https://user-images.githubusercontent.com/67296794/124364509-65317d00-dc18-11eb-8d87-64df50cea728.PNG)

**Comment by giswqs** [comment]
Use the toolbar to open CSV. If it can open the file, that means you are not writing code properly. 

![image](https://user-images.githubusercontent.com/5016453/124365347-601efe80-dc15-11eb-9950-b38a84de2952.png)

**Reply by coblinski** [reply]
Ok, the first problem was the kind of delimiter. I had to change from ; to , in my computer.

Now I could load the csv file in both ways (by code and by toolbox)....but the points didn't display in the map. nothing.

**Reply by giswqs** [reply]
Please provide a small subset of your data for debugging.

**Reply by coblinski** [reply]
I am sending the file in xlsx format, I couldnt upload in csv, I don'tr know why..  so  transform it please.
[test12.xlsx](https://github.com/giswqs/leafmap/files/6759949/test12.xlsx)

**Comment by giswqs** [comment, marked as answer]
Please double check you data. Latitude must be [-90, 90], and longitude must be [-180, 180]. 

![https://i.imgur.com/M6JAeYO.png](https://i.imgur.com/M6JAeYO.png)
